apps/pie-docs/src/_includes/scripts.njk

```diff
@@ -1,4 +1,7 @@
 {# All JS modules to go here #}
+<script src="{{ '/assets/js/external/lit-hydrate-support.js' | url }}" type="module"></script>
+<script src="{{ '/assets/js/external/dsd-polyfill.js' | url }}" type="module"></script>
```

Comment:
can these two scripts use the `defer` attribute too? :) [reference](https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement/defer)